project-koku/koku
# [COST-2960] dont delete ocp on cloud data on ocp resummary

## Jira Ticket

[COST-2960](https://issues.redhat.com/browse/COST-2960)

## Description

This will prevent the data inserted into the OCP summary table via the OCP on Cloud summary flow from being deleted when we resummarize OCP data. This makes the OCP on Cloud summary updater solely responsible for deleting and resummarizing that specific data into the OCP summary table. 💫 😵 

## Testing

1. Checkout main
2. Load test customer data
3. Run psql query 
```select sum(infrastructure_raw_cost) as infrastructure_raw_cost from reporting_ocpusagelineitem_daily_summary where usage_start between '2022-08-01' and '2022-08-31';```
4. Run report_data masu API for OCP cluster running on AWS http://127.0.0.1:5042/api/cost-management/v1/report_data/?provider_uuid=20731a07-fb4b-4f34-b82a-441ff0d79665&schema=org1234567&start_date=2022-08-01&end_date=2022-08-31
5. Run same query from 3 and notice the reduction in total cost
6. Run report_data masu API for the AWS cluster to recalculate OCP on AWS cost and reinsert into OCP daily summary table
7. Run same query from 3 and notice cost matches original again.
8. Check out branch `cost-2960-dont-delete-ocp-on-cloud-data-on-ocp-resummary`
9. Run same report_data call as in 4.
10. Run same query from 3 and notice cost still matches original.

## Notes

...
